packages/autorest.typescript/test/smoke/generated/graphrbac-data-plane/review/graphrbac-data-plane.api.md

```diff
@@ -283,7 +283,7 @@ export interface DirectoryObject {
     [property: string]: any;
     readonly deletionTimestamp?: Date;
     readonly objectId?: string;
-    objectType: "User" | "Application" | "Group" | "ServicePrincipal";
+    objectType: string;
```

Comment:
This is expected. DirectoryObject has four subclasses, "User" , "Application",  "Group" , "ServicePrincipal", but they didn't specify x-ms-discriminator-value, in this case the discriminator value for this four subclasses should have the value of the  class name itself. but that doesn't mean objectType can only have this four values.